### PR TITLE
[all] (index.yaml) Adding links to main landing page

### PR DESCRIPTION
**Pull request by alison-mk**
This PR includes: 
-- my attempt at adding a paragraph to the main landing page (mostly unsuccessful because the page schema is very restrictive -- doesn't allow ":" characters, couldn't figure out how to do a line break, etc). Any suggestions here?? 
-- since the paragraph didn't work, I experimented with adding MS Learn links to the About Office Add-ins card. If we include MS Learn links here, my intuition is that they should open in a new tab, and also that there be some visual indicator that these thinks go to a different section of the site (a small icon?). Not sure how much freedom we have. Open to suggestions here, too!

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[eaa6707](https://github.com/OfficeDev/office-js-docs-pr/commits/eaa670798f082c45b40c390c922c264e93ea657c)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/index.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/alison-mk-landing-intros/docs/index.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/index?branch=pr-en-us-2058) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C8%5C1%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202008010035062851-2058%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=NLOFHmTLXKc0iwzbPPtrQ6%2FY0%2B9TwFTgLKt5gWL%2FOLI%3D&st=2020-08-01T00%3A31%3A32Z&se=2020-09-01T00%3A36%3A32Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/index.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ### YamlMime:Landing
 
 title: Office Add-ins documentation # < 60 chars
-summary: Use the Office Add-ins platform to build solutions that extend Office applications and interact with content in Office documents. With Office Add-ins, you can use familiar web technologies such as HTML, CSS, and JavaScript to build solutions that can run in Office on the web, Windows, Mac, and iPad. # < 160 chars
+summary: Use the Office Add-ins platform to build solutions that extend Office applications and interact with content in Office documents. With Office Add-ins, you can use familiar web technologies such as HTML, CSS, and JavaScript to build solutions that can run in Office on the web, Windows, Mac, and iPad. To build your first add-in within a few minutes, check out the Quick starts. To take a deeper dive into add-in development, explore the Tutorials. For an interactive learning experience, try the Microsoft Learn Office Add-in modules, like Introduction to Office client customization with add-ins and Extend Office clients with Office add-ins. # < 160 chars
```
Comment (AlexJerabek) [inline docs/index.yml:4]: The comment at the end seems to imply that this value has to be fewer than 160 characters. Is that not true?

File: docs/index.yml [hunk 2]
```diff
@@ -38,6 +38,12 @@ landingContent:
             url: project/index.yml
           - text: Word add-ins documentation
             url: word/index.yml
+      - linkListType: learn
+        links: 
+          - text: Introduction to Office client customization with add-ins
+            url: //docs.microsoft.com/learn/modules/intro-office-add-ins/
```
Comment (AlexJerabek) [inline docs/index.yml:44]: Since this is a docs.microsoft.com link, you don't need anything before the ".com".
```suggestion
            url: /learn/modules/intro-office-add-ins/
```

File: docs/index.yml [hunk 3]
```diff
@@ -38,6 +38,12 @@ landingContent:
...
+          - text: Extend Office clients with Office add-ins
+            url: //docs.microsoft.com/learn/paths/m365-office-add-in-associate/
```
Comment (AlexJerabek) [inline docs/index.yml:46]:
```suggestion
            url: /learn/paths/m365-office-add-in-associate/
```

File: docs/index.yml [hunk 4]
```diff
@@ -38,6 +38,12 @@ landingContent:
             url: project/index.yml
           - text: Word add-ins documentation
             url: word/index.yml
+      - linkListType: learn
```
Comment (ElizabethSamuel-MSFT) [inline docs/index.yml:41]: Maybe we can move this learn section higher - place after concept?

File: docs/index.yml (hunk 1 above)
Comment (ElizabethSamuel-MSFT) [reply docs/index.yml:4]: The original was already longer than 160 - maybe we consider this a suggestion, not a hard-and-fast rule...
Landing page documentation: https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-landing-page?branch=master

File: docs/index.yml (hunk 3 above)
Comment (alison-mk) [reply docs/index.yml:46]: Nifty! I didn't realize we could easily link to other doc sets!

**Comment by o365devx** [comment]
Docs Build status updates of commit _[31445e0](https://github.com/OfficeDev/office-js-docs-pr/commits/31445e0335e7de9cfa170c959f03d8342a67f493)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/index.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/alison-mk-landing-intros/docs/index.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/index?branch=pr-en-us-2058) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C8%5C3%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202008032335445755-2058%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=3251IMdMKzPSSW2KXswI5Wnsz3zyAarhGUBHi898Gyc%3D&st=2020-08-03T23%3A32%3A09Z&se=2020-09-03T23%3A37%3A09Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by o365devx** [comment]
Docs Build status updates of commit _[f94f12a](https://github.com/OfficeDev/office-js-docs-pr/commits/f94f12a4445065a626889de8cf5360cb33d1e0d9)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/index.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/alison-mk-landing-intros/docs/index.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/index?branch=pr-en-us-2058) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C8%5C3%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202008032336310824-2058%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=J3SrZekP54vSSrP1pkgp2OTBnf5hTEa2EWbbS%2BFwkec%3D&st=2020-08-03T23%3A33%3A40Z&se=2020-09-03T23%3A38%3A40Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/index.yml (hunk 1 above)
Comment (alison-mk) [reply docs/index.yml:4]: Thank you for this link, Elizabeth! I think this paragraph is too long as is (it's inaccessible and hard to parse), so I only want to add my new content if there's a way to break it into two paragraphs. Based on the documentation + my testing last week, I don't think that's possible. I also originally envisioned including a link to the Learn module here, and that doesn't seem possible in the summary field either.

File: docs/index.yml (hunk 4 above)
Comment (alison-mk) [reply docs/index.yml:41]: I like that idea! All of the documentation links under Overview are also available in the sidebar, so I don't think we lose much by moving them down.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[cd2f292](https://github.com/OfficeDev/office-js-docs-pr/commits/cd2f2925151611e9009226dde4a82d76a04bcec7)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/404.md](https://github.com/OfficeDev/office-js-docs-pr/blob/alison-mk-landing-intros/docs/404.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/404?branch=pr-en-us-2058) | [Details](#user-content-9684adf0-81dde7eb)
[docs/index.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/alison-mk-landing-intros/docs/index.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/index?branch=pr-en-us-2058) |

<a id="9684adf0-81dde7eb"></a>
### [docs/404.md](https://github.com/OfficeDev/office-js-docs-pr/blob/alison-mk-landing-intros/docs/404.md)
 - **Line 0, Column 0**: **[Warning-custom-404-page]** `````Custom 404 page will be deprecated in future. Please remove the 404.md file to resolve this warning.`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C8%5C4%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202008042213305890-2058%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=2fTutYZXV5TXkn%2BcTi%2FZ5mfKStwOwDEIayo3Yoa%2BXVs%3D&st=2020-08-04T22%3A09%3A12Z&se=2020-09-04T22%3A14%3A12Z&sp=r).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://teams.microsoft.com/l/message/19:7ecffca1166a4a3986fed528cf0870ee@thread.skype/1590030602688?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&parentMessageId=1590030602688&teamName=Docs%20Support&channelName=General&createdTime=1590030602688) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by o365devx** [comment]
Docs Build status updates of commit _[b1c7095](https://github.com/OfficeDev/office-js-docs-pr/commits/b1c7095a4bbfd00e83d4b205cf50edced705ee76)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/404.md](https://github.com/OfficeDev/office-js-docs-pr/blob/alison-mk-landing-intros/docs/404.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/404?branch=pr-en-us-2058) | [Details](#user-content-9684adf0-bd836ad4)
[docs/index.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/alison-mk-landing-intros/docs/index.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/index?branch=pr-en-us-2058) |

<a id="9684adf0-bd836ad4"></a>
### [docs/404.md](https://github.com/OfficeDev/office-js-docs-pr/blob/alison-mk-landing-intros/docs/404.md)
 - **Line 0, Column 0**: **[Warning-custom-404-page]** `````Custom 404 page will be deprecated in future. Please remove the 404.md file to resolve this warning.`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C8%5C4%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202008042216129069-2058%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=BcfHyS0WXSA93%2BAQtyDFDmlMmu9Y1L%2B%2BXVZRj%2BwOA60%3D&st=2020-08-04T22%3A12%3A03Z&se=2020-09-04T22%3A17%3A03Z&sp=r).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://teams.microsoft.com/l/message/19:7ecffca1166a4a3986fed528cf0870ee@thread.skype/1590030602688?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&parentMessageId=1590030602688&teamName=Docs%20Support&channelName=General&createdTime=1590030602688) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by alison-mk** [comment]
Thanks @AlexJerabek!
